operations/workflow.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+  
+BRANCH_PATH="$HOME/branch/"
+
+while getopts ":b :c" opt; do
+  case $opt in
+    b)
+      read -p "Enter the branch label: " branch
```

Comment:
Can you check if the branch exists on the remote beforehand allowing the user to proceed?

with this command:

```
git ls-remote --exit-code --heads git@github.com:user/repo.git branch-name
```

exit code of 2 would tell you if there was no matching branch on the remote (halt or require input again).
exit code of 0 would mean it does exist (proceed).